source-salesforce/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "source-salesforce"
+version = "0.1.0"
+description = ""
+authors = ["Luishfs <luish1980@hotmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.11,<3.12"
+flow-sdk = {path="../python", develop=true}
```

Comment:
This is using the older flow-sdk, which we've now replaced with the estuary-cdk.

Switching everything over has been a work in progress that's happened concurrently with your efforts here ([see this just-landed PR](https://github.com/estuary/connectors/pull/1326)). Would you please update accordingly?

Reply:
modified in [ae13497](https://github.com/estuary/connectors/pull/1355/commits/ae134972e57190b963389f75daf5e7d6a1bf6d92)

Reply:
Modified in [a1c34e9](https://github.com/estuary/connectors/pull/1355/commits/a1c34e924c00ab8a950e247e7bff669e8fcd0abd)